resources/tentacle.rb

```diff
@@ -18,7 +18,7 @@
 # limitations under the License.
 #
 
-actions :install, :configure, :remove, :uninstall
+actions :install, :configure, :register, :remove, :uninstall
```

Comment:
Done in a01581ed64acc1f557dbb19c13a2a4005a378684
All new resource attributes have verify methods. Which other methods were needed?